requirements.txt

```diff
@@ -10,3 +10,6 @@ zarr
 fsspec
 s3fs
 xarray-datatree==0.0.6
+psutil==5.9.1
+pandas==1.4.3
```

Comment:
I think xarray already brings in pandas, so you probably don't need to declare this.

Reply:
Yes, you are correct that xarray brings in pandas. I will go ahead and remove this.